OC4IDS_Indicator_Coverage.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<p>In this plot there is an unlabelled bar. I think this might be the missing 'value for money' use case which is depreciated due to being a duplicate but I wasn't sure?!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
You can see it here https://colab.research.google.com/drive/1MFt35_MeoqTOX5ewtpLvwBMzmDXwUB6V?usp=sharing

Reply:
<p>The summary omits deprecated indicators, but one of the other indicators lacked a category. I was waiting for feedback from Ben on removing it, but I've deprecated it for now. The chart should no longer have an unlabelled bar.</p>